syntaxes/tacl.tmLanguage.json

```
@@ -335,7 +343,7 @@
         "numbers": {
             "patterns": [
                 {
-                    "match": "(?<!\\^)[+-]?[0-9]+\\b",
+                    "match": "(?<!\\^)(\\s)[+-]?[0-9]+\\b",
```

```suggestion
                    "match": "(?<!\\^)\\b[+-]?[0-9]+\\b",
```
Match word boundary instead of white space.